src/js/button.js

```diff
@@ -70,7 +70,7 @@ class Button extends Component {
   // KeyPress (document level) - Trigger click when keys are pressed
   onKeyPress(event) {
     // Check for space bar (32) or enter (13) keys
-    if (event.which == 32 || event.which == 13) {
+    if (event.which === 32 || event.which === 13) {
```

Comment:
We should be careful about changes like this that aren't directly related and touch a lot of files. We've got a hack week going on right now and master is going to change a lot, so there's going to be a lot of merge conflicts.